src/App.css

```diff
@@ -2472,8 +2512,17 @@ code, .code {
   margin-left: 3px;
 }
 
-.SplitPane.animating .Pane {
-  transition: width .4s ease;
+.function > .thought-container > .thought > .editable {
+  font-family: monospace;
+}
+
+.function > .thought-container > .thought-annotation > .subthought {
+  background-color: #ddd;
```

Comment:
I think you coul define your color tokens in a central place such that it would be easy to change themes, should that be a requirement. What do you think?

Reply:
Totally agree